x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -1349,7 +1349,6 @@
     "data.mgmt.searchSessions.status.expiresSoonInHoursTooltip": "{numHours} heures",
     "data.mgmt.searchSessions.status.message.createdOn": "Expire le {expireDate}",
     "data.mgmt.searchSessions.status.message.expiredOn": "Expiré le {expireDate}",
-    "data.painlessError.painlessScriptedFieldErrorMessage": "Erreur d'exécution du champ d'exécution ou du champ scripté sur le modèle d'indexation {indexPatternName}",
```

Comment:
I think we're still missing some of these renames. Since we haven't changed the text, we should probably just rename the keys for the translation files instead of removing them altogether:

* `data.painlessError.painlessScriptedFieldErrorMessage`
* `data.errors.fetchError`
* `data.esError.unknownRootCause`
* `data.painlessError.buttonTxt`
* `data.search.esErrorTitle`
* `data.search.httpErrorTitle`